### PR TITLE
Code Review Request for Project 2 Milestone2

### PR DESCRIPTION
**Pull request by amiraabouhadid**
Please review our code for the second milestone of the second project in the Microverse 2.0 program. We were asked to clone the Next Web webpage and use media query to resize it for three different screen sizes: mobile, tablet (min-width:768px), and desktop (min-width:1024px). In the second milestone, we added the categories, latest news, and latest deals sections and formatted them accordingly to the tablet and desktop screens.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello  👋

**You've done pretty well here ensuring all sections in this milestone are responsive. 👌**

However, let's make a couple of changes.

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: style.css [hunk 1]
```diff
@@ -0,0 +1,1128 @@
...
+  height: 5rem;
+}
+
+#ln-articles-container article {
+  display: flex;
+  width: 100%;
+  position: relative;
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+  padding-bottom: 1rem;
+}
```
Comment (mikenath223) [inline style.css]: - [x] Please avoid using id selectors for styling, it's unconventional and is a [css bad practice](https://speckyboy.com/good-bad-css-practices/),. Kindly replace the id selectors in this milestone with class selectors.

File: README.md [hunk 2]
```diff
@@ -5,9 +5,9 @@ Project 2: Building with Responsive Design
 
 Project 2 of Microverse 2.0 program: Building with Responsive Design
 
-We were asked to make a clone of The New World page using media query to work with 3 screen sizes: mobile, tablet, and desktop using mediaqueries.
+We were asked to make a clone of The Next Web page using media query to work with 3 screen sizes: mobile, tablet, and desktop using mediaqueries.
 
-![READMESNEAKPEAK](https://user-images.githubusercontent.com/56790126/108226304-75532f00-7145-11eb-9f71-7916903fbc67.png)
```
Comment (mikenath223) [inline README.md]: - [x] On the readme, please change the term `Project Name` to the actual name of the project.

File: style.css (hunk 1 above)
Comment (amiraabouhadid) [reply style.css]: The ids used in css file were approved in previous milestone.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello  👋

**You've done pretty well here fixing a few issues raised in the previous review. 👌**

However, let's make a couple of changes.

- It seems you erroneously deleted `.stylelintrc.json`, kindly add it again. It's important you follow the linting rules stated in this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) as those rules are shared by other students too.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Review by umairarshadbutt — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @amiraabouhadid ! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
**If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Pull request merged**